re2/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.0)
 project(re2 CXX C)
 
+add_definitions(-Wno-narrowing)
```

Comment:
Yes, but we don't build with GCC 5 yet for a lot of other reasons. And this change only turns off a warning, it does not fix a compilation issue. And the compilation issue was fixed elsewhere anyways :)